fastplotlib/layouts.py

```diff
@@ -111,9 +119,19 @@ def __init__(
                 renderer=renderer
             )
 
+            self.indexers[i, j] = GridPlotIndexer((i,j))
+
         self._animate_funcs: List[callable] = list()
         self._current_iter = None
 
+    def __getitem__(self, index: Union[tuple[int, int], str]):
```

Comment:
I think capital T for `Tuple` in the type annotation is required for python < 3.10